packages/office-ui-fabric-react/src/components/Icon/Icon.base.tsx

```diff
@@ -40,7 +40,8 @@ export class IconBase extends BaseComponent<IIconProps, IIconState> {
           'aria-label': ariaLabel
         }
       : {
-          role: 'presentation'
+          role: 'presentation',
+          'aria-hidden': true
```

Comment:
My only concern is if there are scenarios where we don't want the icon to be hidden. I'm leaning towards no, even if the icon was an image, it would probably be hidden and not read out loud even if there was an alt tag. 

If this is true we should add some quick documentation stating that icons are only decorative, and will be hidden from the UIA tree.

Reply:
What about simply adding a check for isImage before setting it to false/true?

`'aria-hidden': isImage ? false : true`

And then moving container props before native props

`<RootType data-icon-name={iconName} {...containerProps} {...nativeProps} className={classNames.root}>`

and fixing components/scenarios case by case?  E.g: CalendarDay.tsx, adding an aria-hidden={false} to the Icon component being used as a button.

Reply:
If an `alt` tag were provided, or `aria-labeledby`, would we still want role/aria-hidden attributes?

Reply:
yeah, i like the idea of removing aria-hidden if an alt tag is provided.

Reply:
Updated.  I moved `containerProps` before `nativeProps` which I think should take precedence over, and set aria-hidden to true **if** `aria-labelledby` or `alt` were defined.

Does it make sense for Icon to keep `role=presentation` for all instances?  I feel like this should only be defined if `alt`, `aria-labelledby`, etc were **not** defined.